dotNetInstallerLib/ProcessComponent.cpp

```diff
@@ -40,6 +40,11 @@ bool ProcessComponent::IsExecuting() const
 
 void ProcessComponent::Wait(DWORD /* tt */)
 {
+    if (m_process_handle == NULL)
+    {
+        return;
+    }

```

Comment:
I will address this in a subsequent PR.
Some of the tests in dotNetInstallerUnitTests.dll cause an exception when run in the Release configuration, but not in the Debug configuration, specifically on the appveyor CI build machine, and for some reason m_process_handle is NULL in this part of the code.
Thank you.